css-canvas-medical/index.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Canvas Medical</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+    <header>
+      <div class="row" id="row1">
+        <div class="header-img">
+          <img src="images/logo-blue.svg" alt="logo-blue.svg">
+        </div>
+        <div class="column-half">
+          <div class="row" id="header-link">
+            <div><a href="">Product</a></div>
+            <div><a href="">Implementation</a></div>
+            <div><a href="">About Us</a></div>
+            <div><a href="link-a">Blog</a></div>
+            <div><a href="" id="button-header">Contact Us</a></div>
+          </div>
+        </div>
+      </div>
+    </header>
+    <main>
+      <div class="row">
+        <div class="column-full" id="row2">
+          <div class="background">
+            <div class="overlay"></div>
+            <h2>The EMR Reimagined for <br> Primary Care</h2>
+            <p>Can't remember the last time you talked with a patient without focusing <br>
+              on the EMR? Canvas lets you rediscover joy in the work you do. Capture <br>
+              meaningful data, identify gaps in care, and move seamlessly between <br>
+              tasks within the platform. You stay present with patients while your <br>
+              whole care team stays up-to-speed. Be a better doctor with Canvas, <br>
+              designed for primary care providers with the help of primary care <br>
+              providers.</p>
+            <button>Get Started</button>
+          </div>
+        </div>
+      </div>
+      <div class="row" id="row3">
```

Comment:
Do not create numbered ids or class names, it is lazy. Create names that accurately describe what they represent.